extensions/wikia/CommunityPage/styles/components/_todo-list-module.scss

```diff
@@ -31,24 +31,21 @@ $edit-pen-url: '/extensions/wikia/CommunityPage/images/edit-pen-gray.svg';
 		width: $edit-icon-size;
 	}
 
-	.community-page-todo-list-module-description {
+	%community-page-todo-list-module-placeholder {
+		border-style: solid;
+		border-left: thick solid $color-buttons;
 		clear: both;
+		margin-top: 10px;
 	}
 
 	.community-page-todo-list-module-content {
-		border-style: solid;
-		border-left: thick solid $color-buttons;
-		margin-top: 10px;
-		max-height: 500px;
-		overflow: auto;
+		@extend %community-page-todo-list-module-placeholder;
 		padding: 0 10px;
 	}
 
 	.community-page-todo-list-module-zerostate {
-		border-style: solid;
-		border-left: thick solid $color-buttons;
+		@extend %community-page-todo-list-module-placeholder;
 		font-weight: bold;
-		margin-top: 10px;
 		padding: 5px 10px;
```

Comment:
some spacing values have 5 or 10px - could we move it to const. ie. `$padding-base: 5px;` or `$padding-small: 5px; $padding-big: 10px;` ?